core/src/main/java/com/scalar/db/api/AbacAdmin.java

```diff
@@ -399,55 +396,48 @@ default List<NamespacePolicy> getNamespacePolicies() throws ExecutionException {
   }
 
   /**
-   * Applies the given policy to the given table of the given namespace.
+   * Creates a table policy with the given policy and the given table.
    *
+   * @param tablePolicyName the table policy name
    * @param policyName the policy name
    * @param namespaceName the namespace name
    * @param tableName the table name
    * @throws ExecutionException if the operation fails
    */
-  default void applyPolicyToTable(String policyName, String namespaceName, String tableName)
+  default void createTablePolicy(
```

Comment:
Renamed `applyPolicyToTable` to `createTablePolicy`.

Also, changed the signature of the `enableTablePolicy`, `disableTablePolicy `, and `getTablePolicy` methods to specify the table policy name.